compiler/codegen/FrontEnd.hpp

```diff
@@ -135,11 +135,14 @@ class TR_FrontEnd : public TR_Uncopyable
 
    virtual TR_OpaqueMethodBlock * getMethodFromName(char * className, char *methodName, char *signature, TR_OpaqueMethodBlock *callingMethod=0);
    virtual uint32_t offsetOfIsOverriddenBit();
+   virtual uint32_t offsetOfIsBreakpointedBit();
 
    // Needs VMThread
    virtual bool isMethodEnterTracingEnabled(TR_OpaqueMethodBlock *method);
    virtual bool isMethodExitTracingEnabled(TR_OpaqueMethodBlock *method);
 
+   virtual bool isSelectiveMethodEnterExitEnabled();
```

Comment:
I would say any new APIs added to the common frontend class ought to be Doxygen documented for precise specification so that any downstream project wishing to implement their own frontend knows exactly how to implement these APIs.